SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularityMailer.java

```diff
@@ -125,7 +132,7 @@ public void stop() throws Exception {
 
     final String fullPath = String.format("%s/%s", directory.get(), filename);
 
-    final Long logLength = Long.valueOf(maybeSmtpConfiguration.get().getTaskLogLength());
+    final Long logLength = (long)maybeSmtpConfiguration.get().getTaskLogLength();
```

Comment:
Mainly because IntelliJ was telling me that it was unnecessarily boxing it, although I just ran some tests and it looks like this is marginally slower than what was there originally.